index.js

```diff
@@ -0,0 +1,6 @@
+/* jshint node: true */
+'use strict';
+
+module.exports = {
+  name: 'model-fragments'
```

Comment:
Sorry, I don't follow the concern here. @slindberg are you saying for consistency reasons or do you think that this will cause trouble with ember-cli? Are there any other packages that have a different name like this?

In terms of what the export name should be, I don't have a strong opinion either way. I do think the name of addon should be "ember-data-model-fragments". So then the question becomes should the module name match. 

@jakesjews FWIW I did use this package in my app without any issue ... but I didn't do `npm install ...` or `ember install ...`. I just included it in my package.json, eg: `"ember-data-model-fragments": "lytics/ember-data-model-fragments#addonify`. I'm not sure if that makes a difference.